l5kit/l5kit/environment/envs/l5_env.py

```diff
@@ -340,11 +341,7 @@ def _get_kin_rescale_params(self, max_num_scenes: int = 10) -> KinematicActionRe
         :param max_num_scenes: maximum number of scenes to consider to determine parameters
         :return: Tuple of the action un-normalization parameters for kinematic model
         """
-        scene_ids = list(range(self.max_scene_id)) if not self.overfit else [self.overfit_scene_id]
-        if len(scene_ids) > max_num_scenes:  # If too many scenes, CPU crashes
-            scene_ids = scene_ids[:max_num_scenes]
-        sim_dataset = SimulationDataset.from_dataset_indices(self.dataset, scene_ids, self.sim_cfg)
-        return calculate_kinematic_rescale_params(sim_dataset)
+        return KinematicActionRescaleParams(math.radians(20) * 0.1, 0.6)
```

Comment:
Sorry I forgot to mention it earlier, can we move these hard coded constants here to global constants in this module ?

Reply:
In which file should I define the global constants?